ArchiSteamFarm/Commands.cs

```diff
@@ -1932,8 +1932,8 @@ internal sealed class Commands {
 				return null;
 			}
 
-			(bool success, string output) = Actions.Exit();
-			return FormatStaticResponse(success ? output : string.Format(Strings.WarningFailedWithError, output));
+			Utilities.InBackground(ASF.RestartOrExit);
```

Comment:
You've noticed this redirects to wrong action, and you didn't think for a second that just redirecting to proper `Restart()` action that already exists is the right way to go about it? 🤔